apps/examples/dtls_client/dtls_client_main.c

```diff
@@ -40,14 +40,32 @@
  * Included Files
  ****************************************************************************/
 
-#include "tls/config.h"
+#include <tinyara/config.h>
```

Comment:
You're right, but it has no harm to include it just for safety. Anyway, I will fix it.